ui/mirage/handlers/mfa-login.js

```diff
@@ -93,11 +95,7 @@ export default function (server) {
     } else if (user === 'mfa-j') {
       [mfa_constraints, methods] = generator([m('pingid')]); // use to test push failures
     }
-    const numbers = (length) =>
-      Math.random()
-        .toString()
-        .substring(2, length + 2);
-    const mfa_request_id = `${numbers(8)}-${numbers(4)}-${numbers(4)}-${numbers(4)}-${numbers(12)}`;
+    const mfa_request_id = `${uuidv4({ rng: rng_browser })}`;
```

Comment:
```suggestion
    const mfa_request_id = crypto.randomUUID()
```

Reply:
I'd prefer using `uuid/v4` rather than bare `crypto.randomUUID` because of wider browser/platform support.

Refs:
- [1] https://developer.mozilla.org/en-US/docs/Web/API/Crypto/getRandomValues#browser_compatibility
- [2] https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID#browser_compatibility

Reply:
Third variant is to add `uuid@9.0.0` to dependencies (unfortunately I'm not familiar with this - point me how to do this) and use `uuid/v4` with native support for `crypto.randomUUID()` (where it's available).

Reply:
I was unable to use `uuid` in preferred way (detect browser capabilities in runtime), so we'll just use `crypto.randomUUID()` as is.

Reply:
Since this is a mirage file and only used for testing, we prefer to use the built-in `randomUUID` rather than import another dependency for this. Separately, we [support](https://developer.hashicorp.com/vault/docs/browser-support) chrome, firefox, safari, microsoft edge, all of which support this method.

Reply:
Haha, i thought `ui/mirage` is built into final artifacts. :sweat_smile: